src/Managers/MediaManager.php

```diff
@@ -127,7 +133,7 @@ public function uploadFromBase64(
 
         $media = $fileAdder->toMediaCollection($collection);
 
-        return $media;
+        return $this->setCustomProperties($media, $customProperties);
```

Comment:
This should happen before `toMediaCollection` is called to reduce queries.

Because the same applies to `uploadFromUrl` I think we should take a slightly different approach in this manager. The value type specific methods should only call and return a `FileAdder` so a method higher up can set the custom properties and call `toMediaCollection`.